src/routing/NotAuthenticatedRoute.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router-dom';
+import { useAuthenticationContext } from '../hooks';
+
+export const NotAuthenticatedRoute = ({ children, ...rest }) => {
+  const { isAuthenticated } = useAuthenticationContext();
+
+  if (isAuthenticated) return <Redirect to={'/'} />;
```

Comment:
This is not redirecting to login/sign up page.
I am unsure how will this behave in unison with the redirect from kratos.

Reply:
The idea here is to protect routes that shouldn't be accessed when you are Authenticated. For example the '/login'. This functionality was implemented months ago.